rubocop.yml

```diff
@@ -18,6 +18,7 @@ AllCops:
   Exclude:
   - "vendor/**/*"
   - "db/schema.rb"
+  - "spec/**/*"
```

Comment:
oh :( ok we can close this PR then and just do it on a project by project
basis I guess

On Thu, Aug 6, 2015 at 12:56 PM, rubyDoomsday <notifications@github.com>
wrote:

> In rubocop.yml
> <https://github.com/MotivationScience/ruby-style-guide/pull/3#discussion_r36438026>
> :
>
> > @@ -18,6 +18,7 @@ AllCops:
> >    Exclude:
> >    - "vendor/**/*"
> >    - "db/schema.rb"
> > +  - "spec/**/*"
>
> The only downside to this, is that it will be rendered useless when
> applied to Validic-Test since all files reside in the spec folder.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MotivationScience/ruby-style-guide/pull/3/files#r36438026>
> .
>



-- 
Mark Phelps
Senior Software Engineer
http://validic.com